src/components/toolbar/js/toolbarController.js

```diff
@@ -0,0 +1,206 @@
+angular
+  .module('material.components.toolbar')
+  .controller('MdToolbarController', MdToolbarController);
```

Comment:
Let's prefix the controller with `$mdToolbarController`, to avoid complications and to be consistent with other internal controllers.